tst/testinstall/ffe.tst

```diff
@@ -118,6 +130,62 @@ gap> Subfields( GF(81) );
 gap> Subfields( GF(2^6) );
 [ GF(2), GF(2^2), GF(2^3), GF(2^6) ]
 
+#
+# test + and * with FFEs and rationals
+#
+gap> odds:=[5/3, 5, (5/3)^100, 5^100];;
+gap> evens:=[4/3, 4, (4/3)^100, 4^100];;
+
+#
+gap> ForAll(odds, x -> x + Z(2) = 0*Z(2));
+true
+gap> ForAll(odds, x -> Z(2) + x = 0*Z(2));
+true
+gap> ForAll(evens, x -> x + Z(2) = Z(2));
+true
+gap> ForAll(evens, x -> Z(2) + x = Z(2));
+true
+
+#
+gap> ForAll(odds, x -> x * Z(2) = Z(2));
+true
+gap> ForAll(odds, x -> Z(2) * x = Z(2));
+true
+gap> ForAll(evens, x -> x * Z(2) = 0*Z(2));
+true
+gap> ForAll(evens, x -> Z(2) * x = 0*Z(2));
+true
+
+#
+gap> LargeGaloisField(4,1);
```

Comment:
Is LargeGaloisField intended to "public". As far as I can see it's called from GaloisField when needed and it's not documented.  It looks to me like it's part of the implementation of GaloisField, and should be tested as such, in case someone wants to change that implementation.

Reply:
`LargeGaloisField` is not "public", but I also don't think this matter: we test many non-public functions in our .tst files, and I think this is well and indeed required in many cases. Of course if the behavior of `LargeGaloisField` were to be changed, then now one needs to adjust the tests -- but I don't think that's a major hurdle (and very unlikely to happen anyway).

Many people would argue that having (unit) tests for virtually every function in your program, including all "internal" ones, is a good thing, because despite being undocumented to the public, they still have an *internal* API, and we are very much interested in catching regressions or other bugs in them.

Reply:
OK. Fair enough.